libraries/mapr_utils.rb

```diff
@@ -0,0 +1,54 @@
+module Mapr
+  # Module for configure.sh script
+  module ConfigureSh
+    module_function
+
+    MAPR_CONFIGURE_SCRIPT = '/opt/mapr/server/configure.sh'.freeze unless const_defined?(:MAPR_CONFIGURE_SCRIPT)
+
+    # Build the configure.sh command depending on args Hash
+    # args Hash should in a { 'args' => value} format
+    # Example : args = { '-N' => "cluster_name", '-C' => "host1,host2,host3", '-Z' => "host4,host5,host6" }
+    # Options with no value are passed like { '-secure' => true }
+    def build_command(args)
+      cmd_string = MAPR_CONFIGURE_SCRIPT.dup
+      args.sort.each do |a, v|
+        # non-value options
+        if v == true
+          cmd_string << " #{a}"
+          next
+        end
+        cmd_string << " #{a} #{v}"
```

Comment:
please consider quoting `v` using `Shellwords.escape`

Reply:
is there any case of the order of option being important?  In this case you might want to use an array

Reply:
The order doesn't matter.

I don't understand your point about escaping.